modules/seqexec/server/src/main/scala/seqexec/server/altair/Altair.scala

```diff
@@ -41,6 +41,8 @@ class Altair[F[_]: Sync] private (controller: AltairController[F],
          Ngs(true) => true
     case _         => false
   }
+
+  def isFollowing: F[Option[Boolean]] = controller.isFollowing
```

Comment:
I needed this method to access this value from TCS, because even if the following state is in Altair, the command to change the state is in TCS